packages/number-line/configure/src/ticks.jsx

```diff
@@ -1,49 +1,205 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
-import { MiniField } from './number-text-field';
-import { tickUtils } from '@pie-element/number-line';
+import { MiniField } from './number-text-field-custom';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
 import * as math from 'mathjs';
 
+export const TickIntervals = ['I', 'F', 'D'];
```

Comment:
please add some comments to make it easier to understand what does I, F and D means. But my suggestion would be to not abbreviate and use the entire world for this. Based on the UI, I assume it refers to Integer, Fraction and Decimal. In this way the configuration would be easier to understand

Reply:
Refactored to use whole word instead of abbreviations. `TickIntervals = ['Interval', 'Fraction', 'Decimal'];`